README.md

```diff
@@ -20,7 +20,7 @@ The development of the supervisor is a bit tricky. Not difficult but tricky.
 
 - You can use the builder to build your supervisor: https://github.com/home-assistant/hassio-build/tree/master/builder
 - Go into a HassOS device or VM and pull your supervisor.
-- Set the developer modus on updater.json
+- Turn on the developer modus in the `updater.json` file (usually `/usr/share/hassio/updater.json`) by setting `channel` to `dev`
```

Comment:
There is an API call for it. The file can be differents places. Maybe we can maintain the API call and a small comment to `updater.json` inside hassio data folder?

Reply:
Oh, you mean POST `/supervisor/options`? Yeah that would also be possible. Though I personally find editing a file quicker than having to type out a full curl command (+ finding the ip address of the VM/machine)